client/gutenberg/extensions/mailchimp/index.js

```diff
@@ -9,32 +9,45 @@ import './editor.scss';
 
 export const name = 'mailchimp';
 
-export const fields = [
-	{ id: 'title', label: __( 'Title' ) },
-	{ id: 'email_placeholder', label: __( 'Placeholder' ) },
-	{ id: 'submit_label', label: __( 'Submit button label' ) },
-	{ id: 'consent_text', label: __( 'Consent text' ) },
-	{ id: 'processing_label', label: __( '"Processing" status message' ) },
-	{ id: 'success_label', label: __( 'Success status message' ) },
-	{ id: 'error_label', label: __( 'Error status message' ) },
-];
-
 export const settings = {
 	title: __( 'MailChimp' ),
 	icon: 'email',
 	category: 'jetpack',
-	keywords: [ __( 'email' ), __( 'mailchimp' ), 'jetpack' ],
-
-	edit,
-	attributes: fields.reduce( ( attrs, field ) => {
-		attrs[ field.id ] = {
+	keywords: [ __( 'email' ), __( 'mailchimp' ) ],
+	attributes: {
+		title: {
 			type: 'string',
-			default: '',
-		};
-		return attrs;
-	}, {} ),
-
-	save: function() {
-		return null;
+			default: __( 'Join my email list' ),
+		},
+		emailPlaceholder: {
+			type: 'string',
+			default: __( 'Enter your email' ),
+		},
+		submitLabel: {
+			type: 'string',
+			default: __( 'Join My Email List' ),
+		},
+		consentText: {
+			type: 'string',
+			default: __(
+				'By clicking submit, you agree to share your email address with the site owner and MailChimp to receive marketing, updates, and other emails from the site owner. Use the unsubscribe link in those emails to opt out at any time.'
+			),
+		},
+		processingLabel: {
+			type: 'string',
+			default: __( 'Processing...' ),
```

Comment:
Ellipsis over three dots:
```suggestion
			default: __( 'Processing…' ),
```

There's an eslint rule for this in `translate` calls, we should bring it over to `__` and related functions now that we have them.